service-runner/src/main/resources/config/assets.yaml

```diff
@@ -4,7 +4,7 @@ assets:
     code: USDC
     issuer: GDQOE23CFSUMSVQK4Y5JHPPYK73VYCNHZHA7ENKCV37P6SUEO6XQBKPP
     distribution_account: GBN4NNCDGJO4XW4KQU3CBIESUJWFVBUZPOKUZHT7W7WRB7CWOA7BXVQF
-    significant_decimals: 2
+    significant_decimals: 4
```

Comment:
Should be 7 FYI

Reply:
It is weird to use `7` in the reference server. WDYT we modify the reference to use `4` for the reason of "reference".